src/open_inwoner/templates/cms/plugins/zaken/zaken.html

```diff
@@ -1,46 +1,30 @@
 {% load i18n icon_tags button_tags django_htmx %}
-{% if request.htmx %}
-    {% if zaken %}
-        <div class="card-container card-container--columns-2 plugin-card">
+{% if show_zaken_plugin %}
+    {% if request.htmx %}
+        <oip-zaken-plugin-container>
+            {% if error_message %}<div slot="error">{{ error_message }}</div> {% endif %}
             {% for zaak in zaken %}
-                {% url "cases:case_detail" object_id=zaak.uuid api_group_id=zaak.api_group.id as zaak_url %}
-                <a href="{{ zaak_url }}" class="card card--status card--status--info">
-                    <div class="userfeed__marker userfeed__marker--info"></div>
-                    <div class="card__body card__body--tabled">
-                        <h3 class="utrecht-heading-3 userfeed__title">Status: {{ zaak.current_status }} {% if zaak.case_type %}- Soort: {{ zaak.case_type }}{% endif %}</h3>
-                        {% include "components/StatusIndicator/StatusIndicator.html" with status_indicator=info status_indicator_text="Let op!" %}
-                        <p class="userfeed-card__description">
-                            <span class="status">{{ zaak.description }}</span>
-                        </p>
-                        <span class="button button--icon-before button--transparent">
-                            {% icon icon="east" icon_position="after" primary=True outlined=True %}
-                        </span>
-                    </div>
-                    <p class="userfeed-card__description">
-                        <span class="status">{{ zaak.identification }}</span>
-                    </p>
-                </a>
+                <oip-zaken-plugin-zaak-item
+                    description="{{ zaak.description }}"
+                    identificatie="{{ zaak.identification }}"
+                    detail-url="{{ zaak.url }}"
+                    >
+                    <div slot="continue-section"></div>
+                </oip-zaken-plugin-zaak-item>
             {% endfor %}
-        </div>
+        </oip-zaken-plugin-container>
     {% else %}
-        <p class="userfeed-card__description">{% trans "U heeft op dit moment geen lopende zaken." %}</p>
-    {% endif %}
-{% else %}
-    {% if instance %}
-        <section class="plugin userfeed">
-            <header class="userfeed__header">
-                <div class="heading-2__indicator">
-                    <h2 class="utrecht-heading-2">{{ instance.title }}</h2>
+        <oip-homepage-plugin-section
+            title="{{ plugin_title }}"
+            next-url="{{ mijn_zaken_url }}"
+            next-url-label="Bekijk alle zaken">
+                <div hx-get="{{ hx_get_url }}">
```

Comment:
This won't do anything on pageload without an HTMX trigger, 
like `hx-trigger="load"` and `hx-swap="outerHTML"`
but i've already added that to my PR that i have rebased on this one, so changes will just conflict.